packages/tool-cache/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@actions/tool-cache",
-	"version": "1.1.1",
+	"version": "1.1.2",
```

Comment:
The version of `tool-cache` was bumped, even though it seems that none of the files in `packages/tool-cache` was changed. Is this expected?

Should it mean to reflect that `tool-cache` is updated to use this new feature of `exec`, I'd expect files such as `packages/tool-cache/src/tool-cache.ts` to be modified:

https://github.com/actions/toolkit/blob/5c894298f2f24441718eb049215aa6decd507193/packages/tool-cache/src/tool-cache.ts#L175

https://github.com/actions/toolkit/blob/5c894298f2f24441718eb049215aa6decd507193/packages/tool-cache/src/tool-cache.ts#L203

https://github.com/actions/toolkit/blob/5c894298f2f24441718eb049215aa6decd507193/packages/tool-cache/src/tool-cache.ts#L239

https://github.com/actions/toolkit/blob/5c894298f2f24441718eb049215aa6decd507193/packages/tool-cache/src/tool-cache.ts#L254